src/components/search-panel/SearchPanel.jsx

```diff
@@ -28,15 +27,35 @@ const Styles = styled.div`
 }
 `;
 
+let name;
+let start;
+let end;
+let child;
+let adult;
+let suit;
+function getValueFromSearch(city) {
+    name = city
+}
 
 function Search(props) {
     return (<div>
         <input placeholder={props.placeholder}
-            className='searchPanel' />
+            className='searchPanel'
+            onChange={city => getValueFromSearch(city.target.value)} />
     </div>
     );
 };
 
+function getStart(st) {
```

Comment:
What does this function do?

Reply:
callback func to get date from "checkIn". We will rename it

Reply:
weird function